modules/security-network-isolating.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * security/container_security/security-network.adoc
+
+[id="security-network-isolating_{context}"]
+= Isolating applications
+
+{product-title} enables you to segment network traffic on a single cluster to
+make multitenant clusters that isolate users, teams, applications, and
+environments from non-global resources.
+
+For example, to isolate a project network in the cluster, run:
+
+----
+$ oc adm pod-network isolate-projects <project1> <project2>
```

Comment:
this part is confusing me,  above command only for `multitenant` plugin mode.  for `multitenant`,  project1 and project 2 are isolated by default.